history/characters/59000_dalaranian.txt

```diff
@@ -60,6 +60,29 @@
 	}
 }
 
+59001={
+	name = Khadgar
+	dna = khadgar_dna
+	culture = dalaranian
+	religion = arcane_religion
+	martial=7 diplomacy=8 stewardship=7 intrigue=7 learning=8 # PH
+	trait=education_learning_4 # Wowpedia: "perhaps the most accomplished of all living wizards and one of the most powerful magi in Azerothian history"
+	trait=intellect_good_2 # Same as Antonidas's. Maybe should be higher?
+	trait=chaste # Most mages are, I guess
+	trait=calm # Never seen Khadgar lose his temper, he's usually pretty calm
+	trait=diligent # He's fought against the Legion like 289389 times and always shows up no matter what
+	trait=brave # Ties into the above point
```

Comment:
I think you should add ``trait_magic_good_3`` since Khadgar is clearly potent with magic.

Reply:
> 
> 
> I think you should add `trait_magic_good_3` since Khadgar is clearly potent with magic.

Added in https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2/pull/204/commits/2acf03e0ed21e62f0f452d989fe87fe0d7dfcfba.